### PR TITLE
feat: upgrade to fundamental-styles 0.2.0

### PR DESCRIPTION
**Pull request by jbadan**
### Description

**KNOWN ISSUES** out of scope of this pr:
1. follow ups in FS: https://github.com/SAP/fundamental-styles/wiki/0.2.0-followup-needed
1. Popover arrow is not showing up at all
1. Popover implementation library styles solutions
    - ComboBox, MultiInput, DatePicker, Timepicker, SearchInput: width of overlay
   - DatePicker: Calendar needs padding inside the Popover body
    - TimePicker: padding for `Time`
1. further followups in FR: https://github.com/SAP/fundamental-react/wiki/Fundamental-styles-0.2.0-followup
1. dynamic imports - prop based
1. `Shellbar SearchInput` is open always - we need to rewrite a ton of logic to get this to be hidden smoothly. 

[BREAKING CHANGE]

#### Infrastructure
1.  Consumers will need their webpack configurations to handle css

#### Components
Changes _besides_ classnames listed here: https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes
1. `ActionBar`
     *  removed `mobile`, `width` props 
1. `Button`
      * removed `dropdown`, `navbar` props
      * removed `fd-dropdown__control`, `fd-global-nav__btn` classes
1. `Calendar`
    * removed `fd-button--standard` class from buttons
1. `ComboBoxInput`
   * removed `fd-combobox-control` class + `div`
1. `FormItem`
   * removed `fd-form-item--check`
   * removed `isCheck` prop
   * replaced `FormItem` checkbox with `Checkbox` component
1. `InputGroup`
   * removed "search" type - use `SearchInput`
   * removed `searchButtonProps` prop
1. `ListGroupItemCheckbox`
   * replaced internals with Checkbox component
1. `MenuItem`
    * removed `separator` prop (now lives in `MenuList`)
1. `Modal`
   * removed extraneous `div` around actions
1. `MultiInput`
   * removed extraneous `div` around popover control
1. `SearchInput`
   * `searchList` prop propTypes is more explicit - requires array of objects with keys including `text` (required) and `callback`
1. `Shellbar`
   * removed several extraneous `div`s
1. `Tile`
   * removed `rowSpan`, `columnSpan` (handled in `LayoutGrid`)
    * removed  `colorAccent`,`backgroundColor` (handled by helper classes)
1. `TimePicker`
   * removed extraneous `div`

Removed Components:
1. `PanelGrid`
1. `TileGrid`

New Components:
1. `Checkbox`
1. `Link`
1. `LayoutGrid`

fixes #issueid

### CODE REVIEW

File: src/ComboboxInput/ComboboxInput.js [hunk 1]
```diff
@@ -1,47 +1,35 @@
...
+                            compact={compact}
+                            placeholder={placeholder}
+                            type='text' />
+                        <span className='fd-input-group__addon fd-input-group__addon--button'>
+                            <Button
+                                {...buttonProps}
+                                compact={compact}
+                                glyph='navigation-down-arrow'
+                                option='light' />
+                        </span>
```
Comment (greg-a-smith) [inline src/ComboboxInput/ComboboxInput.js]: Shouldn't this just go through the `InputGroup` component?

File: src/DatePicker/DatePicker.js [hunk 2]
```diff
@@ -323,9 +318,11 @@ class DatePicker extends Component {
                                 type='text'
                                 value={this.state.formattedDate} />
                             <span className='fd-input-group__addon fd-input-group__addon--after fd-input-group__addon--button'>
```
Comment (greg-a-smith) [inline src/DatePicker/DatePicker.js]: Shouldn't this just go through the `InputGroup` component?

File: src/Dropdown/Dropdown.Component.js [hunk 3]
```diff
@@ -14,173 +14,164 @@ export const DropdownComponent = () => {
...
-                            id='jhqD0561'
-                            noArrow />
-                    </Dropdown>
-                </div>
+                            </Button>
+                        }
+                        disabled
+                        id='jhqD0561'
+                        noArrow />
+                </Dropdown>
```
Comment (greg-a-smith) [inline src/Dropdown/Dropdown.Component.js]: We never actually changed `Dropdown` yet, have we?

File: src/Dropdown/Dropdown.test.js [hunk 4]
```diff
@@ -45,7 +45,7 @@ describe('<Dropdown />', () => {
             <Popover
                 body={defaultMenu}
                 control={
-                    <Button dropdown type='standard'>
+                    <Button dropdown>
```
Comment (greg-a-smith) [inline src/Dropdown/Dropdown.test.js]: Didn't the `dropdown` prop get removed from `Button`?

File: src/Badge/Counter.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import 'fundamental-styles/dist/badge.css';
```
[the remark this answers is not in the document]
Comment (jbadan) [reply src/Badge/Counter.js]: nope

**Pull request merged**